From 282826292c2f432f5de07f808a496c9f84f9ce38 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 18 Nov 2024 15:03:58 +0100
Subject: [PATCH 1373/1695] dts: stm32: rename scmi regulators by regulator@0
 for stm32mp25x

Use a scmi regulator node name coherent with scmi binding:
protocol@17{
	regulators {
		regulator@0 {
		};
		[...]
	};
};

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f87ae5a9a01b8b61482fbbfc260cbda237a72ff9 dts: stm32: rename scmi regulators by regulator@0 for stm32mp25x boards
043da248993f5d985fb08f51f96e33e933b82f2f dts: stm32: rename scmi regulators by regulator@0 for stm32mp25x]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I021918547ddcec8577160948cf0f996a264209b6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419306
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index b30986254..a2600ac33 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -1164,31 +1164,31 @@
 			scmi_regulator: protocol@17 {
 				reg = <0x17>;
 
-				scmi_regu: regulators {
+				regulators {
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					voltd-vddio1 {
+					regulator@0 {
 						reg = <VOLTD_SCMI_VDDIO1>;
 						voltd-supply = <&vddio1>;
 					};
 
-					voltd-vddio2 {
+					regulator@1 {
 						reg = <VOLTD_SCMI_VDDIO2>;
 						voltd-supply = <&vddio2>;
 					};
 
-					voltd-vddio3 {
+					regulator@2 {
 						reg = <VOLTD_SCMI_VDDIO3>;
 						voltd-supply = <&vddio3>;
 					};
 
-					voltd-vddio4 {
+					regulator@3 {
 						reg = <VOLTD_SCMI_VDDIO4>;
 						voltd-supply = <&vddio4>;
 					};
 
-					voltd-vdda18adc {
+					regulator@7 {
 						reg = <VOLTD_SCMI_ADC>;
 						voltd-supply = <&vdda18adc>;
 					};
@@ -2209,7 +2209,7 @@
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					voltd-vdd33ucpd {
+					regulator@5 {
 						reg = <VOLTD_SCMI_UCPD>;
 						voltd-supply = <&vdd33ucpd>;
 					};
-- 
2.47.2

